app/jobs/onceoff/fix_encoded_category_slugs.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Jobs
+
+  class FixEncodedCategorySlugs < ::Jobs::Onceoff
+    def execute_onceoff(args)
+      return unless SiteSetting.slug_generation_method == 'encoded'
+
+      #Make custom categories slugs nil and let the app regenerate with proper encoded ones
+      Category.where('id > 4').each do |c|
```

Comment:
`id > 4`? why that specific hard coded value?

Reply:
I want to skip the built-in categories (staff, uncategorized, Lounge, Site Feedback are 1,2,3,4). Is there a way to query those equivalent to `User.human?` or `Group.automatic?` ?

Reply:
It doesn't look like there is. I think at least we should use a constant here. Perhaps add a `seeded?` method to `Category` that is true if the `id` is in that range. Later we could make that an actual column.

Reply:
I agree. It would be a good idea to add `seeded?` and check if the `id` equals one of the following site settings.

https://github.com/discourse/discourse/blob/48c123849228cc8a392e989558fe89734cd02656/config/site_settings.yml#L1823-L1835

Reply:
Damn this is so much better! Thanks for the suggestion @gschlager !